pkg/api/login.go

```diff
@@ -283,3 +323,11 @@ func (hs *HTTPServer) RedirectResponseWithError(ctx *models.ReqContext, err erro
 
 	return Redirect(setting.AppSubUrl + "/login")
 }
+
+func (hs *HTTPServer) SendLoginLog(cmd *models.SendLoginLogCommand) {
+	if err := bus.Dispatch(cmd); err != nil {
+		if err != bus.ErrHandlerNotFound {
+			hs.log.Warn("Error while sending login log", "err", err)
+		}
```

Comment:
Won't this complain for everything all the time when you're running OSS?

Reply:
No, the error will be `bus.ErrHandlerNotFound` so we won't log it.